Gruntfile.js

```diff
@@ -8,27 +8,15 @@ module.exports = function (grunt) {
 
   // Node and client side JS have slightly different JSHint directives
   // We'll create 2 versions with .jshintrc as a baseline
-  var browserJSHint = grunt.file.readJSON('.jshintrc');
-  var nodeJSHint = grunt.file.readJSON('.jshintrc');
-
-  // Don't throw errors for expected Node globals
-  nodeJSHint.node = true;
+  var JSHint = grunt.file.readJSON('node_modules/mofo-style/linters/.jshintrc');
```

Comment:
Interesting.

So, doing this it seems globals must be defined inside the .jshintrc, cannot add them later doing it this way.

See: https://github.com/gruntjs/grunt-contrib-jshint

"If a filename is specified, options and globals defined therein will be used"

Because it's a string and not an object at the time we have it, we cannot simply add to it.

We can add globals to the mofo-style repo, but it does add a significant overhead every time we want to introduce a new global, not something I would agree to.

This didn't work:

```
var JSHint = {
  jshintrc: 'node_modules/mofo-style/linters/.jshintrc',
  globals: {
    angular: false,
    $: false,
    define: false,
    requirejs: false
  }
};
```

Not sure why, but seems jshintrc as a filename just overrides anything else.